packages/component-library/src/components/plmg-status/plmg-status.tsx

```diff
@@ -0,0 +1,89 @@
+import { Component, h, Watch, Prop } from '@stencil/core';
+import { PlmgStatusVariant, isPlmgStatusVariant } from './plmg-status.types';
+
+@Component({
+  tag: 'plmg-status',
+  styleUrl: 'plmg-status.scss',
+  shadow: true,
+})
+export class Status {
+  /**
+   * Define status variant
+   *
+   * Allowed values:
+   *   - neutral
+   *   - danger
+   *   - info
+   *   - warning
+   *   - success
+   *
+   * Sets background and text colors
+   *
+   * required
+   */
+  @Prop() variant!: PlmgStatusVariant;
+  @Watch('variant')
+  validateVariant(newValue: string) {
+    if (!isPlmgStatusVariant(newValue))
+      throw new Error('variant: must be a valid value');
```

Comment:
Inconsistent error message format. The others do not use a colon. 
```suggestion
      throw new Error('variant must be a valid value');
```